snowplow_tracker/snowplow.py

```diff
@@ -0,0 +1,124 @@
+# """
+#     snowplow.py
+
+#     Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
+
+#     This program is licensed to you under the Apache License Version 2.0,
+#     and you may not use this file except in compliance with the Apache License
+#     Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+#     http://www.apache.org/licenses/LICENSE-2.0.
+
+#     Unless required by applicable law or agreed to in writing,
+#     software distributed under the Apache License Version 2.0 is distributed on
+#     an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#     express or implied. See the Apache License Version 2.0 for the specific
+#     language governing permissions and limitations there under.
+
+#     Authors: Jack Keene, Anuj More, Alex Dean, Fred Blundun, Paul Boocock
+#     Copyright: Copyright (c) 2013-2022 Snowplow Analytics Ltd
+#     License: Apache License Version 2.0
+# """
+import logging
+from typing import  Optional
+from snowplow_tracker import Tracker, Emitter, subject, EmitterConfiguration
+from snowplow_tracker.typing import JsonEncoderFunction
+
+# Logging
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+    
+
+"""
+Snowplow Class
+"""
+class Snowplow:
+    trackers = {}
```

Comment:
Just to make it clear that this is a private variable:

```suggestion
    _trackers = {}
```